tests/migrations/0003_add_test_model_with_schema.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-12-26 11:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+test_model_fields = [
+    ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+    ('int_field', models.IntegerField(null=True, blank=True)),
+    ('name', models.CharField(max_length=50, null=True, blank=True))
+]
```

Comment:
Again, why fields are placed separately? All fields are standard and are present in all django versions